docs/content/en/_index.html

```diff
@@ -38,7 +38,7 @@
             <li>Automated deployment analysis based on metrics, logs, emitted requests</li>
             <li>Automatically roll back to the previous state as soon as analysis or a pipeline stage fails</li>
             <li>Automatically detect configuration drift to notify and render the changes</li>
-            <li>Automatically watch and detect the new container images to deploy</li>
+            <li>Automatically trigger a new deployment when an arbitrary event has occurred (e.g. container image updated, helm chart publishing, etc)</li>
```

Comment:
👍 I do not have a strong opinion on using this `arbitrary` (it's cool 😎) but it missed the meaning of `users have to define/specific what is the event to listen/watch. That why at the suggestion `defined/specified` is provided, but I would like to hear your thought too 😄

Reply:
Yes, "defined/specified" makes me feel better.
And "publishing" -> "published" to be unified.

Reply:
I have no strong opinion on that as well. But anyway I wanted to say that users can automatically trigger a new deployment at any time they want.

Okay before refreshing my browser, I found there are two votes. Let's do so.